src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/register/BitbucketWebhookHandler.java

```diff
@@ -35,12 +32,22 @@
  */
 public class BitbucketWebhookHandler implements WebhookHandler {
 
+    private static final Collection<BitbucketWebhookEvent> ALL_PULL_REQUEST_EVENTS = Arrays.asList(PULL_REQUEST_DECLINED,
+            PULL_REQUEST_DELETED, PULL_REQUEST_FROM_REF_UPDATED, PULL_REQUEST_MERGED, PULL_REQUEST_OPENED);
+    private static final Collection<String> ALL_PULL_REQUEST_EVENT_IDS = new HashSet<>();
```

Comment:
This is only being used once, so it can probably be inlined in the `process` method

Reply:
These fields are actually not used at all any more, I've removed them all